cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -61,7 +61,7 @@ THE SOFTWARE.
         <j:set var="id" value="${h.generateId()}"/>
         <f:block>
             <input type="button" value="Generate Groovy" onclick="handlePrototype_${id}(); return false" class="submit-button primary"/>
-            <f:textarea id="prototypeText_${id}" readonly="true"/>
+            <f:textarea id="prototypeText_${id}" readonly="true" style="margin: 10px 0px;" />
```

Comment:
maybe try wrap with `f.entry() { <f:textarea ... }` that is common pattern in jenkins jelly? AFAIR in comparison to block in adds small indentation vertically